Sources/TuistLoader/Models+ManifestMappers/TargetDependency+ManifestMapper.swift

```diff
@@ -30,44 +30,42 @@ extension TuistGraph.TargetDependency {
         manifest: ProjectDescription.TargetDependency,
         generatorPaths: GeneratorPaths,
         dependenciesGraph: DependenciesGraph
-    ) throws -> TuistGraph.TargetDependency {
+    ) throws -> [TuistGraph.TargetDependency] {
```

Comment:
Could you elaborate about his change? :)

Reply:
Related to [this](https://github.com/tuist/tuist/pull/3099#discussion_r657272802).
One `ProjectDefinition.TargetDependency.external` might be mapped to multiple `TuistGraph.TargetDependency` because an SPM product might contain multiple targets